src/index.js

```diff
@@ -3,11 +3,17 @@ import now from 'performance-now';
 import { defaultFrameDuration } from './constants';
 
 type Stub = {|
-  add: (cb: Function) => number,
+  add: {
```

Comment:
why did we need to duplicate the api under here?

Reply:
Looks like spread doesn't work with `[[call]]` well